docs/language.rst

```diff
@@ -2201,6 +2201,110 @@ not allowed to call functions or read variables in the initializer:
         uint constant byzantium_block = 4_370_000;
     }
 
+
+Binding methods to types with ``using``
+---------------------------------------
+
+The builtin types and the any user-defined types like structs, can be extended
+with methods using the ``using`` syntax. This can be done either using libraries
```

Comment:
```suggestion
We can bind methods to builtin types and any user-defined types like structs using the ``using`` syntax. This can be done either using libraries
```
Please, note that if you decide not to accept this suggestion, the comma placed between "structs" and "can be" should be removed, because we must not split the subject and the verb with a comma.

Reply:
I think in technical writing, it is best to avoid "we" or "you". I've specifically avoided this in the documentation so far.

Reply:
I've seen many technical publications using "we", including research papers. I don't think it is that bad. In addition, I've always heard that we should avoid using passive voice in English. If you want to avoid we, maybe this is a better way to write the phrase:
```suggestion
Custom methods can be bound to builtin types and any user-defined types like structs using the ``using`` syntax. This can be done either using libraries
```

Reply:
In papers "we" means the authors of the paper did something. Here it is unclear who "we" refers to.

Reply:
I've adopted your change without the "custom", thanks

Reply:
Feel free to remove `custom`. This is just a suggestion.